Rakefile

```diff
@@ -28,17 +27,18 @@ desc "Default: Run all examples"
 task :default => :spec
 
 spec = Gem::Specification.new do |s|
+  s.name = "ruby-hl7"
+  s.version = RubyHl7::VERSION
   s.homepage = "https://github.com/ruby-hl7/ruby-hl7"
   s.platform = Gem::Platform::RUBY
   s.summary = "Ruby HL7 Library"
   s.description = "A simple library to parse and generate HL7 2.x messages"
+  s.authors = ["Mark Guzman"]
```

Comment:
Not sure what the recommendations are for this field but it may be good to list yourself and @mogox.